index.js

```diff
@@ -0,0 +1,64 @@
+import addBook from './modules/addBook.js';
+import { DateTime } from './modules/luxon.js';
+
+const booksForm = document.querySelector('.books-form');
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+
+window.onload = () => {
+  const date = document.getElementById('date');
+  date.innerHTML = DateTime.now().toLocaleString(DateTime.DATETIME_HUGE);
+};
```

Comment:
- [ ] [Optional] It would be much better if you edit the date and time format to be just like the given design.
See this [link ](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time)for more info.

| Given Design  | Your Design |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/68796232/183933871-e5ac6860-53f7-417c-8aab-7fa2f7e36edb.png)  | ![image](https://user-images.githubusercontent.com/68796232/183933939-15569245-0f98-424c-b37d-3e84a193a565.png)  |

Reply:
@Omar-Muhamad Great! You are correct. Thank you for the review.